src/index.scss

```diff
@@ -11,7 +11,19 @@
   min-height: 100vh;
 
   main {
+    flex-direction: row !important;
```

Comment:
why you have to set `flex-direction` of main element?

Reply:
main container flex-direction was set to column, to make tabs-nav sidebar, I changed flex-direction to row.

Reply:
main container flex-direction was set to column, to make tabs-nav sidebar, I changed flex-direction to row.